Documentation/troubleshooting.rst

```diff
@@ -293,54 +293,6 @@ Opening the ``status``, we can drill down through ``policy.realized.l4``. Do you
 ``ingress`` and ``egress`` rules match what you expect? If not, the reference to the errant
 rules can be found in the ``derived-from-rules`` node.
 
-Automatic Diagnosis
-===================
-
-The ``cluster-diagnosis`` tool can help identify the most commonly encountered
-issues in Cilium deployments. The tool currently supports Kubernetes
-and Minikube clusters only.
-
-The tool performs various checks and provides hints to fix specific
-issues that it has identified.
-
-The following is a list of prerequisites:
-
-* Requires Python >= 2.7.*
-* Requires ``kubectl``.
-* ``kubectl`` should be pointing to your cluster before running the tool.
```

Comment:
Don't we want to keep the list of requirements in the doc?

Reply:
They are still there, in the section "Automatic log & state collection". The information was duplicated.